src/Streamly/Data/Stream/Concurrent.hs

```diff
@@ -106,6 +106,40 @@ module Streamly.Data.Stream.Concurrent
 where
 
 import Streamly.Internal.Data.Stream.Concurrent
+  ( MonadAsync
+  , Config
+  , StopWhen(..)
+  , Rate(..)
+  , maxThreads
+  , maxBuffer
+  , rate
+  , inspect
+  , eager
+  , stopWhen
+  , ordered
+  , interleaved
+  , avgRate
+  , minRate
+  , maxRate
+  , constRate
+  , fromCallback
+  , parApply
+  , parConcat
+  , parConcatIterate
+  , parConcatMap
+  , parEval
+  , parList
+  , parMapM
+  , parMergeBy
+  , parMergeByM
+  , parRepeatM
+  , parReplicateM
+  , parSequence
+  , parTapCount
+  , parZipWith
+  , parZipWithM
+  , tapCount
+  )
```

Comment:
Why explicit imports when there are so many? Also, these are not from streamly-core.